### PR TITLE
Prototype for <vaadin-message> options menu

### PR DESCRIPTION
**Pull request by anezthes**
If we can, and want to, render `<vaadin-message>` in `<vaadin-message-list>` in Light DOM, this menu should be somewhat accessible.

Followed this example:
https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/menu-button/menu-button-1/menu-button-1.html

This is just a prototype, it'll need some proper development to be production ready. For example, the menu items don't do anything just yet and they are hard-coded.

### CODE REVIEW

File: src/vaadin-message.js [hunk 1]
```diff
@@ -6,7 +6,12 @@
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import '@polymer/iron-icon/iron-icon.js';
```
Comment (web-padawan) [inline src/vaadin-message.js:9]: Please do not use `iron-icon`. We are using a different approach in components.

File: src/vaadin-message.js [hunk 2]
```diff
@@ -6,7 +6,12 @@
...
 import '@vaadin/vaadin-avatar/src/vaadin-avatar.js';
+import '@vaadin/vaadin-button/src/vaadin-button.js';
+import '@vaadin/vaadin-context-menu/src/vaadin-context-menu.js';
+import '@vaadin/vaadin-icons/vaadin-icons.js';
```
Comment (web-padawan) [inline src/vaadin-message.js:13]: Please do not use `vaadin-icons` as we do not use in components either.

File: src/vaadin-message.js [hunk 3]
```diff
@@ -141,8 +156,23 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
...
+            <vaadin-button
+              aria-controls="vaadin-message-actions-menu"
+              aria-expanded="false"
+              aria-haspopup="true"
+              aria-label="Actions"
+              id="vaadin-message-actions-button"
+              theme="icon tertiary-inline"
+            >
+              <iron-icon aria-hidden="true" icon="vaadin:ellipsis-dots-v"></iron-icon>
+            </vaadin-button>
```
Comment (web-padawan) [inline src/vaadin-message.js:174]: Consider using the same DOM structure and styles as [vaadin-menu-bar overflow button](https://github.com/vaadin/vaadin-menu-bar/blob/fd7ebd5d34ca18684b32e72143f844c11fdb8071/src/vaadin-menu-bar.js#L88-L90).

**Review by web-padawan — CHANGES_REQUESTED** [review]

File: src/vaadin-message.js (hunk 1 above)
Comment (anezthes) [reply src/vaadin-message.js:9]: Understood. I'll check what we use in other components.

File: src/vaadin-message.js (hunk 3 above)
Comment (anezthes) [reply src/vaadin-message.js:174]: I'll have a look-see.

**Comment by web-padawan** [comment]
Closing as outdated. Let's open a new PR later, in case if we decide to proceed with this feature.

**Pull request closed without merging**